less/bootstrap-override/type.less

```diff
@@ -105,7 +108,7 @@ h6, .h6 { font-size: @font-size-h6; }
 	color: #757575 !important;
 }
 .text-primary {
-	color: #2075c0 !important;
+	color: #3074af !important;
```

Comment:
I noticed there were `!important` declarations here... should we be doing this??? I added the `.text-danger` declaration above to account for the updated color, but did not add `!important` onto it. It still overrides the bootstrap style as expected. Unless there are solid reasons for doing so, we should avoid `!important` declarations in our LESS.